api/envoy/config/listener/v3/listener_components.proto

```diff
@@ -227,6 +227,11 @@ message FilterChain {
   // connections established with the listener. Order matters as the filters are
   // processed sequentially as connection events happen. Note: If the filter
   // list is empty, the connection will close by default.
+  //
+  // For QUIC listeners, network filters can be created, but due to differences
+  // in the connection implementation compared to TCP, the onData() method will
+  // never be called. Therefore network filters for QUIC listeners should only
+  // expect to do work at the start of a new connection (i.e. in onNewConnection()).
```

Comment:
it would be good to mention, the HCM required to be the last filter for udp.

Reply:
Done, thanks